python_modules/libraries/dagster-databricks/setup.py

```diff
@@ -39,7 +39,7 @@ def get_version() -> str:
         f"dagster-pyspark{pin}",
         "databricks-cli~=0.17",  # TODO: Remove this dependency in the next minor release.
         "databricks_api",  # TODO: Remove this dependency in the next minor release.
-        "databricks-sdk<0.9",  # Breaking changes occur in minor versions.
+        "databricks-sdk==0.17.0",  # dbt-databricks is pinned to this version
```

Comment:
Please adjust to `<=0.17.0`, some of our users may be constrained to older versions.